README.md

```diff
@@ -2,3 +2,20 @@
 a template repository for the 2439 midterm project
 
 [Applied Challenge #1 Instructions](https://docs.google.com/document/d/e/2PACX-1vTbclFb83xohS9hWkVu7YA7JnSVJqh5Lz53HwQE6UONAhgtpvLjYeUGF1lqurwyjg/pub)
+
+Scope Statement:
+
+With this challenge, I hoped to create a functional and
+easy-to-use example web application that could be navigated and
+utilized with little confusion, and that was also pleasant
+on the eyes. I wanted to create the four required pages to
+the best of my ability along with a home page to act as a hub between
+them.
+
+Link to Usability Tests:
+
+https://docs.google.com/document/d/1jkjI1zGxRpr0xxXopD6wn7U4eS34gLKtrpKMcK2ZohA/edit?usp=sharing
```

Comment:
What a great and thorough test from #2! You tried to tackle some functionality beyond the scope of the assignment - which is awesome! You've got some great feedback here!